test/unit/models/viewmodels/RulingViewModelTest.scala

```diff
@@ -27,30 +27,12 @@ class RulingViewModelTest extends UnitSpec {
 
   "showEditRuling" should {
 
-    "show edit ruling button when case status is OPEN" in {
+    "show edit ruling button when we have EDIT_RULING permission" in {
       val aCase = dummyCase.copy(status = OPEN)
       val permissions: Set[Permission] = Set(Permission.EDIT_RULING)
       RulingViewModel.fromCase(aCase, permissions).showEditRuling shouldBe true
     }
 
-    "show edit ruling button when case status is REFERRED" in {
-      val aCase = dummyCase.copy(status = REFERRED)
-      val permissions: Set[Permission] = Set(Permission.EDIT_RULING)
-      RulingViewModel.fromCase(aCase, permissions).showEditRuling shouldBe true
-    }
-
-    "show edit ruling button when case status is SUSPENDED" in {
-      val aCase = dummyCase.copy(status = SUSPENDED)
-      val permissions: Set[Permission] = Set(Permission.EDIT_RULING)
-      RulingViewModel.fromCase(aCase, permissions).showEditRuling shouldBe true
-    }
-
-    "not show edit ruling button because case status" in {
-      val aCase = dummyCase.copy(status = REJECTED)
-      val permissions: Set[Permission] = Set(Permission.EDIT_RULING)
-      RulingViewModel.fromCase(aCase, permissions).showEditRuling shouldBe false
-    }
```

Comment:
These tests are irrelevant now because it's only the permission which determines whether to display the link